### ISSUE TITLE
Install themes using the dashboard?

### ISSUE THREAD

**Issue by d1y** [doki-theme/doki-theme-vscode#102]
I want to use the theme in a visual way, because now I can’t preview it with the command installation, sometimes the theme doesn’t match my expectations

1. add `command`: `doki-theme.dashboard`
2. use vscode extension `webview` panel
![Screen Shot 2021-08-14 at 16 17 57](https://user-images.githubusercontent.com/45585937/129439859-d4fcde81-cbe2-452c-ad7f-80327bf19d71.png)

Probably like this, can the author accept pr, I want to realize this part of the function, thank you for the plug-in, it is very easy to use

**Comment by Unthrottled** [comment]
Okay, I am a bit confused. 

This is what think your problem is:

"There are a lot of stickers and wallpapers, and I do not know what each of them look like before installing theme."

You want a solution that solves that problem, right?

So you envision the `dashboard` as a way to quickly preview and install available images to your vs-code?

**Comment by d1y** [comment]
> Okay, I am a bit confused.
> 
> This is what think your problem is:
> 
> "There are a lot of stickers and wallpapers, and I do not know what each of them look like before installing them."
> 
> You want a solution that solves that problem, right?
> 
> So you envision the `dashboard` as a way to quickly preview and install available images to your vs-code?

Yes, I need to provide a preview interface to let me choose a theme

**Comment by Unthrottled** [comment]
Okay, that makes more sense. Thank you for the suggestion, it will probably be a while before I can work on this. 

In the mean time, you can browse all the themes in the [screenshot album](https://github.com/doki-theme/doki-theme-vscode/blob/master/albums/complete_theme_album.md) or see all the previews here: https://doki-theme.unthrottled.io/themes

Thanks for the suggestion!
